isabelle/haskell/MarloweCoreJson.lhs

```diff
@@ -0,0 +1,1394 @@
+{-
+    This module adds Json encoding and decoding capabilities
+    to the SemanticTypes generated by Isabelle.
+
+    It is written with Literal Haskell to be both the actual code
+    and part of the specification.
+-}
+
+-- To use lhs2TeX we need to include the "template" of tex generation
+%include polycode.fmt
+
+-- We configure ghci to include the relevant files so that \eval can print
+-- them
+%options cabal exec ghci -- -iisabelle/generated:isabelle/haskell
+
+-- The tex that this file generates includes all the necesary directives
+-- to be called by pdflatex directly, but we are actually including this
+-- inside of the specification that Isabelle generates, so we need to
+-- provide a...
+-- ===PATTERN FOR SED===
+
+%if False
+-- Everything inside the `if False` block is not seen by lhs2TeX but is
```

Comment:
I wasn't sure if it was worth while to include this inside the specification... it might be weird to see some weird `T.` or `C.` qualified imports without looking at the code.